pkgs/tools/security/otpgen/default.nix

```diff
@@ -0,0 +1,48 @@
+{ 
+  pkgs
+  , lib
+  , stdenv
+  , makeWrapper
+  , fetchFromGitHub
+  , cmake
+  , pkg-config
+  , cryptopp
+  , qtbase
+}:
+
+stdenv.mkDerivation rec {
+
```

Comment:
Space here is not necessary.

Reply:
Sorry for the dumb question but which space?

Reply:
I mean the empty line.

Reply:
Fixed here https://github.com/NixOS/nixpkgs/pull/87887/commits/f70af1e46c6f7fa18556a83889bde4547972551a

Reply:
Actually, an empty line is still here.